images.CI/macos/anka/Anka.Helpers.psm1

```diff
@@ -22,7 +22,9 @@ function Push-AnkaTemplateToRegistry {
         Invoke-WebRequest -Uri $uri -Method Delete | Out-Null
     }
 
-    $command = "anka registry --registry-path $RegistryUrl push --force --tag $TagName $TemplateName"
+    $AnkaConfigPath="/Users/maccloud/.config/anka/certs"
```

Comment:
I think we can change `/Users/maccloud/` with just `$HOME`

Reply:
Updated thanks